### DISCUSSION TITLE
Which models keep notes local?

### DISCUSSION THREAD

**Discussion by hittjwX** [brianpetro/obsidian-smart-connections#582] [Q&A]
I don't want to send my notes to the cloud. Which model keep my notes inside the Obsidian ecosystem. I have a powerful computer. Terms of service on most AI platforms are concerning.

**Comment by brianpetro** [comment]
Currently, the only cloud embedding models are from OpenAI. All of the other models are local 🌴

**Reply by hittjwX** [reply]
Awesome. I have 22,000 notes. Some of which are client work and commissioned reports. Thanks.
